### PR TITLE
Upgrade DatabaseManager

### PR DESCRIPTION
**Pull request by ramumnesys**
Transfer magic values to properties file
Add comments
Tweak some code

### CODE REVIEW

File: FoosLiveAndroid/FoosLiveAndroid/Assets/app.properties [hunk 1]
```diff
@@ -31,8 +31,11 @@ percentage_of_side = 0.4
 
 connection_url = http://yu-gi-oh.lt/backend.php
 operation_success = SUCCESS
-get_timeout = 15000
-
+db_timeout = 10000
+insert_game_format = InsertGame;%1$s;%2$s
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/Assets/app.properties]: Can you explain the values after InsertGame?

File: FoosLiveAndroid/FoosLiveAndroid/Util/Database/DatabaseManager.cs [hunk 2]
```diff
@@ -4,57 +4,71 @@
...
+        private static readonly string InsertEventFormat = PropertiesManager.GetProperty("insert_event_format");
+        private static readonly string GetHistoryFormat = PropertiesManager.GetProperty("get_history_format");
+       
         /// <summary>
         /// Inserts a game into the remote database. 
         /// </summary>
         /// <param name="blueTeamName"> Blue team name</param>
         /// <param name="redTeamName">Red team name</param>
-        /// <returns>A task that returns int. The int is the id of the inserted game, or -1 if error happens.
+        /// <returns>The id of the inserted game, or null if error happens.
```
Comment (pauliuskalv) [inline FoosLiveAndroid/FoosLiveAndroid/Util/Database/DatabaseManager.cs]: This function doesn't return null, as the documentation says, unless I misunderstood something

File: FoosLiveAndroid/FoosLiveAndroid/Assets/app.properties (hunk 1 above)
Comment (ramumnesys) [reply FoosLiveAndroid/FoosLiveAndroid/Assets/app.properties]: There is no point of explaining it in properties file. This is string format for String.Format(format,  <values>). The format is used to insert values in specific order, manner. https://docs.microsoft.com/en-us/dotnet/standard/base-types/composite-formatting is kind of the same.

Comment (ramumnesys) [reply FoosLiveAndroid/FoosLiveAndroid/Assets/app.properties]: %1$s stand fors 1 argument after format,  $s stands for string.

File: FoosLiveAndroid/FoosLiveAndroid/Util/Database/DatabaseManager.cs (hunk 2 above)
Comment (ramumnesys) [reply FoosLiveAndroid/FoosLiveAndroid/Util/Database/DatabaseManager.cs]: My bad.

**Pull request merged**